### PR TITLE
#86 quit game button logic and UI

### PR DESCRIPTION
**Pull request by n0macx**
-  added Quit Game button with Scss Styles 
-  added appearance of Mode Buttons after quit game active
- after Quit Game returning to Mode Rules view
- after Quit Game timers are stopped and disappear

### CODE REVIEW

**Comment by vieraboschkova** [comment]
image
Play the game shouldnt be visible on the initial screen I think, if it is out of the scope of this task, do you know who is responsible for this button?

**Comment by vieraboschkova** [comment]
image
Should the settings button be visible?

**Comment by vieraboschkova** [comment]
image
After clicking quit game (the text shuold be clickable too) the menu with modes doesnt appear and the wand is still visible. Let me know if its within the scope of this task.

File: static/styles/_quitGame.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.quit-game {
+  //   display: inline;
```
Comment (vieraboschkova) [inline static/styles/_quitGame.scss:2]: Clean the unused code, organize properties and add pointer to the cursor please.

**Review by MonikaKrella — APPROVED** [review]

**Pull request merged**